src/Mealz/MealBundle/Entity/AbstractMessage.php

```diff
@@ -33,7 +33,7 @@ public function isEnabled()
     /**
      * @param boolean $disabled
      */
-    public function setEnabled($disabled)
+    public function setEnabled($disabled): void
```

Comment:
"bool" can be used as type-hint for $disabled.

Reply:
Done.